ng-test/src/app/user.address/address.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { LoaderService, User, UserService } from '../../../../dist'
+
+@Component({
+  selector: 'app-address',
+  templateUrl: './address.component.html',
+  styleUrls: ['./address.component.scss']
+})
+export class AddressComponent implements OnInit {
+
+  user: User = new User();
+
+  constructor(
+    private loaderSrv: LoaderService,
+    private userSrv: UserService
+  ) { }
```

Comment:
## Tu dois repenser  la création de composant utiliser comme une page et les composants utiliser comme un widget
Avec address et card on a un exemple de components différents. 
il faut utiliser les @Input et @Output check the doc ici https://angular.io/guide/component-interaction
en gros 
```html
<div *ngFor="let add of user.addresses"> 
  <app-address [address]="add"></app-address>
...
```